__tests__/app.test.js

```diff
@@ -56,33 +56,69 @@ describe("/api/articles/:article_id", () => {
           expect(article).toEqual({
-            author: 'butter_bridge',
-            title: 'Living in the shadow of a great man',
+            author: "butter_bridge",
+            title: "Living in the shadow of a great man",
             article_id: 1,
-            body: 'I find this existence challenging',
-            topic: 'mitch',
-            created_at: '2020-07-09T20:11:00.000Z',
+            body: "I find this existence challenging",
+            topic: "mitch",
+            created_at: "2020-07-09T20:11:00.000Z",
             votes: 100,
-            article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
-          })    
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
         });
     });
     test("status: 404 responds with appropriate message when given valid but non existent article_id", () => {
       return request(app)
-      .get("/api/articles/10000")
-      .expect(404)
-      .then(({body: {message}}) => {
-        expect(message).toBe("Not Found")
-      })
-    })
+        .get("/api/articles/10000")
+        .expect(404)
+        .then(({ body: { message } }) => {
+          expect(message).toBe("Not Found");
+        });
+    });
     test("status: 400 responds with appropriate message when given invalid article_id", () => {
       return request(app)
-      .get("/api/articles/nonsense")
-      .expect(400)
-      .then((response) => {
-        const message = response.body.message
-        expect(message).toBe("Bad Request")
-      })
-    })
+        .get("/api/articles/nonsense")
+        .expect(400)
+        .then((response) => {
+          const message = response.body.message;
+          expect(message).toBe("Bad Request");
+        });
+    });
+  });
+});
+
+describe("/api/articles", () => {
+  describe("GET", () => {
+    test("status: 200 responds with array of articles objects with correct properties", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(Array.isArray(articles)).toBe(true);
+          expect(articles.length).toBeGreaterThan(0);
+          articles.forEach((article) => {
+            expect(typeof article.author).toBe("string");
+            expect(typeof article.title).toBe("string");
+            expect(typeof article.article_id).toBe("number");
+            expect(typeof article.topic).toBe("string");
+            expect(typeof article.created_at).toBe("string");
+            expect(typeof article.votes).toBe("number");
+            expect(typeof article.article_img_url).toBe("string");
+            expect(typeof article.comment_count).toBe("number");
+            expect(typeof article.body).toBe("undefined");
+          });
+        });
+    });
```

Comment:
Good test, great to see you checking the length of the array.

We want to make sure that there isn't a `body` property, so either you don't need the line 110, or could do something check that article does not have the body property, instead of checking it's `undefined`